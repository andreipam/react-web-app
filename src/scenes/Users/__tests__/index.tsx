import * as React from 'react';
import { Route } from 'react-router';
import createRouterContext from 'react-router-test-context';

import { shallow } from 'enzyme';

import { UsersWithoutHOC } from '../index';

describe('(Scene) Users', () => {
  it('renders without crashing', () => {
    const context = createRouterContext();
    const wrapper = shallow(
      <Route path="/" render={props => <UsersWithoutHOC {...props} />} />,
      { context }
    );
    expect(wrapper.dive()).toMatchSnapshot();
  });
});
