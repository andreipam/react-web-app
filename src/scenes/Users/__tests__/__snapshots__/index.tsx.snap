// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Scene) Users renders without crashing 1`] = `
<React.Fragment>
  <h2>
    User overview
  </h2>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="//:id"
    />
  </Switch>
</React.Fragment>
`;
